### PR TITLE
Update docs for parsing

### PR DESCRIPTION
**Pull request by Mletter1**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  pseudoxml   to make pseudoxml-XML files for display purposes
+  linkcheck   to check all external links for integrity
+  doctest     to run all doctests embedded in the documentation (if enabled)
+  coverage    to run coverage check of the documentation (if enabled)
+  clean       to remove everything in the build directory
+  ```
+
+# Adding the docs to the slycat webserver build
+
+You will need to copy the built docs from `/docs/_build` to ~TBA
```
Comment (Spurs20) [inline docs/README.md]: ~TBA needs to be updated to the correct path.

Comment (Mletter1) [reply docs/README.md]: Weird I think I updated this in [f9d1b5b](https://github.com/sandialabs/slycat/pull/1149/commits/f9d1b5b360d1a22cf2c72907d01b373065f82be7)

Comment (Mletter1) [reply docs/README.md]: Oh I see the outdated label on this thread so maybe you were working with the older version.

File: docs/source/PUT-methods/PUT-Project-CSV-Data.rst
Comment (Spurs20) [inline docs/source/PUT-methods/PUT-Project-CSV-Data.rst:1]: This description doesn't match the web service functionality. It was probably added to the wrong doc on accident.

File: web-client/slycat/__init__.py [hunk 2]
```diff
@@ -2,4 +2,4 @@
 # DE-NA0003525 with National Technology and Engineering Solutions of Sandia, LLC, the U.S. Government
 # retains certain rights in this software.
 
-__version__ = "3.1.1"
+__version__ = "3.4.2"
```
Comment (Mletter1) [inline web-client/slycat/__init__.py:5]: correct for tag this commit will become

**Pull request merged**